tests/Aspire.Npgsql.Tests/ConformanceTests.cs

```diff
@@ -35,7 +35,8 @@ public class ConformanceTests : ConformanceTests<NpgsqlDataSource, NpgsqlSetting
 
     protected override bool SupportsKeyedRegistrations => true;
 
-    protected override bool CanConnectToServer => s_canConnectToServer.Value;
+    protected override bool CanCreateClientWithoutConnectingToServer => false;
```

Comment:
Why do we need this change?

Reply:
This property defaults to `true`. Due to that:
https://github.com/dotnet/aspire/blob/16679d35f9ab64d7221033888a94ec495780aead/tests/Aspire.Components.Common.Tests/ConformanceTests.cs#L464-L470

.. if you have `ConnectToServer=false` - no docker available - then it wouldn't skip the test and instead fail for some tests. Now that I think about it, this should be used in tests that don't depend on really connecting to the server. So, either it is incorrectly used in some test or I missed something.

I ran the tests again, and they pass even if this is `true`. In fact, we have this for redis also but isn't needed. Looking at it, IIUC I blindly copied this bit from your rabbitmq PR, which does need this setting!